src/core/transactionmanager.js

```diff
@@ -520,6 +521,17 @@ TransactionManager.prototype._resubmit = function(ledgers_, pending_) {
 TransactionManager.prototype._prepareRequest = function(tx) {
   const submitRequest = this._remote.requestSubmit();
 
+  if (tx.hasMultiSigners()) {
+    tx.setSigningPubKey('');
+
+    if (this._remote.local_signing) {
+      tx.tx_json.Signers = tx.getMultiSigners();
+    } else {
+      submitRequest.message.command = 'submit_multisigned';
+      submitRequest.message.Signers = tx.getMultiSigners();
+    }
+  }
+
   if (this._remote.local_signing) {
     tx.sign();
```

Comment:
Scott, I think Brandon was able to 'submit' a tx_blob with both
TxnSignature AND the Signers array.